### DISCUSSION TITLE
Conditional Element Display AND OR operators

### DISCUSSION THREAD

**Discussion by wfdigitalsupport** [Budibase/budibase#5934] [Help] [answered]
Hi all, 

Can anyone assist with explaining how I can achieve the following in budibase?

Scenario: I want to display a date picker if a user selects one or both options in a multi-select option. If one option is selected from the multi-select only one date picker is shown, but if both options are selected two date pickers are shown for the user to set both dates. 

I've tried using the display if the option is selected condition but both date pickers does not show. 

Any suggestions? 

P.s. no javascript skills 😢

**Comment by melohagan** [comment, marked as answer]
Hi @wfdigitalsupport 

This can be done with conditional UI: https://docs.budibase.com/docs/conditions

For each of your date pickers, you can *Configure conditions* like so:
<img width="979" alt="Screenshot 2022-05-17 at 15 50 45" src="https://user-images.githubusercontent.com/101575380/168840984-2920b8ce-79b8-4c52-bd85-480ca78db8c9.png">

And the JavaScript is reasonably simple:
```
return $("New Form.Fields.Multi").includes('Option A') ? "show" : "hide"
```

The binding will be your multi-select field. 
Then for the second date picker, you would enter 'Option B' or whatever the two options are.

**Reply by wfdigitalsupport** [reply]
hey @melohagan worked a treat!

thanks mate.
